src/dataset.rs

```diff
@@ -30,22 +30,105 @@ use crate::raster::Group;
 
 use bitflags::bitflags;
 
-/// A 2-D affine transform mapping pixel coordiates to world
-/// coordinates. See [GDALGetGeoTransform] for more details.
+/// A six-element array storing the coefficients of an [affine transform]
+/// used in mapping coordinates between pixel/line `(P,L)` (raster) space,
```

Comment:
```suggestion
/// used in mapping coordinates between pixel/line `(P, L)` (raster) space,
```

(same for the other places)